### PR TITLE
fix develop command on windows

### PR DESCRIPTION
**Pull request by prog**
## Description

`gatsby develop` command is broken on windows now

There are backslashes in `developProcessPath`, what should be properly escaped.

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Thanks for the fix!

Bummer we missed this.

Could you replace it with our internal `slash` command that we use for this? https://github.com/gatsbyjs/gatsby/blob/b6dd67d77a7135663d17aa4c70048a1ce0b36681/packages/gatsby-core-utils/src/path.ts#L86

example usage: https://github.com/gatsbyjs/gatsby/blob/2cefdcb1cd41efad48e9f09710f404398be49d3b/packages/gatsby/src/utils/tracer/index.ts#L1

**Comment by KyleAMathews** [comment]
I don't have a windows machine to test this on so would appreciate testing before or after I release it. It's a safe change but I'm not 100% sure it fixes the underlying problem.

**Comment by TechGuyTN** [comment]
I'm watching and will apply fix as soon as tests finish. What would be the proper way to apply this fix on my machine? @KyleAMathews

File: packages/gatsby/src/commands/develop.ts [hunk 1]
```diff
@@ -102,14 +102,14 @@ class ControllableScript {
...
-  const developProcessPath = require.resolve(`./develop-process`)
+  const developProcessPath = slash(require.resolve(`./develop-process`))
   // Run the actual develop server on a random port, and the proxy on the program port
   // which users will access
   const proxyPort = program.port
   const developPort = await getRandomPort()
 
   const developProcess = new ControllableScript(`
-    const cmd = require("${developProcessPath}");
+    const cmd = require(developProcessPath);
```
Comment (pieh) [inline packages/gatsby/src/commands/develop.ts:112]:
```suggestion
    const cmd = require("${developProcessPath}")
```
need to revert this - this is inside template string literal

**Comment by KyleAMathews** [comment]
@TechGuyTN the easiest/fastest way is directly edit `node_modules/gatsby/dist/commands/develop.js` to make the change and try running develop again

**Comment by TechGuyTN** [comment]
Thank you @prog :D :D

**Pull request merged**